yarn.lock

```diff
@@ -12285,15 +12528,14 @@ signal-exit@^3.0.0, signal-exit@^3.0.2, signal-exit@^3.0.3, signal-exit@^3.0.7:
   resolved "https://registry.yarnpkg.com/signal-exit/-/signal-exit-3.0.7.tgz#a9a1767f8af84155114eaabd73f99273c8f59ad9"
   integrity sha512-wnD2ZE+l+SPC/uoS0vXeE9L1+0wuaMqKlfz9AMUo38JsyLSBWSFcHR1Rri62LZc12vLr1gb3jl7iwQhgwpAbGQ==
 
-simple-plist@2.0.0-rc.0, simple-plist@^1.0.0:
-  version "2.0.0-rc.0"
-  resolved "https://registry.yarnpkg.com/simple-plist/-/simple-plist-2.0.0-rc.0.tgz#d3518aadbfbea1a28a7b1495cd5fdd5c1041828e"
-  integrity sha512-838j6M5V7LQLZ+vIEbEA/FaOmL/FiHpWPE/1+c4iZg3n/gHcTawetzVHCBFLMlEYE7k9OUOi/VNoIeMyN7MKYA==
+simple-plist@^1.0.0:
```

Comment:
Uhh.... this would be bad since it would reintroduce a security issue. Why did this file need to be regenerated?

Reply:
It kind of looks like the main package.json's resolutions are not being respected, since it looks like metro 0.66 is also being added even though we should be on 0.64.

Reply:
I think we hard coded a resolution to metro 0.66?

Reply:
Yep: https://github.com/microsoft/fluentui-react-native/blob/457a7eb89d12823f045c7ebf5194487159a79ba6/package.json#L62

Reply:
Ah okay yeah, I was confused, seems like 0.64 is being added to our yarn.lock even though the resolution should force 0.66.

Reply:
Looks like the file is now not changing, so we should be good now.